packages/launchpad/cypress/e2e/global-mode.cy.ts

```diff
@@ -314,4 +317,19 @@ describe('Launchpad: Global Mode', () => {
       })
     })
   })
+
+  describe('error state', () => {
+    it('should not persist the error when going back to the main screen projects', () => {
+      cy.openGlobalMode()
+      cy.addProject('config-with-import-error')
+      cy.addProject('todos')
+      cy.visitLaunchpad()
+      cy.contains('[data-cy="project-card"]', 'todos').should('be.visible')
+      cy.contains('[data-cy="project-card"]', 'config-with-import-error').should('be.visible')
```

Comment:
Can add `.click()` to the end of this line and remove the line below

Reply:
Ah yeah I was hitting a detached element but it was due to another version